hedera-node/src/main/java/com/hedera/services/state/virtual/ContractKey.java

```diff
@@ -33,7 +33,7 @@
 import java.util.Arrays;
 import java.util.stream.Collectors;
 
-import static com.swirlds.jasperdb.utilities.NonCryptographicHashing.perm64;
+import static com.swirlds.common.utility.NonCryptographicHashing.hash32;
```

Comment:
Did somethinhg change for hashing ?

Reply:
Just the name of the library changed.  👍